src/Mod/PartDesign/App/FeatureExtrude.cpp

```diff
@@ -684,7 +684,11 @@ App::DocumentObjectExecReturn* FeatureExtrude::buildExtrusion(ExtrudeOptions opt
                 return App::DocumentObject::StdReturn;
             }
             try {
-                prism.makeElementPrismUntil(base,
+                TopoShape _base;
```

Comment:
Very rarely is this form of TopoShape appropriate; usually we want `TopoShape _base(0);`

However this point may be moot anyway as investigation continues on this bug.

Reply:
We only want to trigger the `isNull()`  test in `makeElementPrismUntil` so it should be ok.